### DISCUSSION TITLE
BLE - write long characteristic value

### DISCUSSION THREAD

**Discussion by amitsi76** [Mair/esp32-course#85] [Course Qeustions] [answered]
Is there any example to show how to write long characteristic value using Nimble stack? I have gone through the BLE course but did not find any reference related to long characteristic value.

**Comment by Mair** [comment, marked as answer]
Hi @amitsi76 
please see https://github.com/espressif/esp-idf/issues/5724

**Reply by amitsi76** [reply]
Thanks a ton. The information was very useful and provided required clarification.
